foreman/foreman.spec

```diff
@@ -424,6 +436,8 @@ cp config/settings.yaml.example config/settings.yaml
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
 %{scl_rake} assets:precompile:all RAILS_ENV=production --trace
+%{scl_rake} db:migrate RAILS_ENV=production --trace
```

Comment:
Just for the record, I was building discovery from 2.0 stable branch and overlooked that apipie initialization patch was removed from the stable branch (remains in the develop)